old-ui/app/config.js

```diff
@@ -234,41 +240,46 @@ ConfigScreen.prototype.componentWillUnmount = function () {
   this.props.dispatch(actions.displayWarning(''))
 }
 
-function rpcValidation (newRpc, state) {
+ConfigScreen.prototype.rpcValidation = function (newRpc, state) {
+  this.setState({
```

Comment:
Minor comment: maybe you can put this inside the `if` and remove the second `setState({ loading: false})`, since the `else` branch is entirely synchronous as far as I can tell.

Reply:
Sure, thanks, fixed https://github.com/poanetwork/nifty-wallet/pull/209/commits/81178e94e91a425d857873a10f17ade440f44341